### PR TITLE
Buildable Reinforced glass + Metal shutters.

### PR DESCRIPTION
**Pull request by Frost-wood**
Pretty simple addition, allowing the player to build reinforced glass, and add openable/closeable metal shutters to the glass, like some convenience stores/mall stores have(or my local library), allowing for a view without a stray zombie destroying that said view.

Uses same amount of reinforced glass that is used in car windows,  Having made reinforced panes from scratch before, it's not an fast and easy process, so the making of this should be relatively balanced.

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
Nice!

**Comment by Rivet-the-Zombie** [comment]
I see both states for the shuttered version have a ````"str_min"```` of 45 versus the base reinforced window's 40. Shouldn't the open version have 40, and the shuttered version be something higher like 60?

**Comment by Frost-wood** [comment]
fixed, didn't think of that

File: data/json/terrain.json [hunk 1]
```diff
@@ -1004,6 +1004,50 @@
         }
     },{
         "type" : "terrain",
+        "id" : "t_reinforced_glass_shutter",
+        "name": "reinforced glass with closed shutters",
+        "symbol": "LINE_OXOX",
+        "color": "ltgray",
+        "move_cost": 0,
+        "flags": ["NOITEM", "WALL", "AUTO_WALL_SYMBOL", "OPENCLOSE_INSIDE"],
+		"open": "t_reinforced_glass_shutter_open",
```
Comment (illi-kun) [inline data/json/terrain.json]: please replace TABs by spaces here.

File: data/json/terrain.json [hunk 2]
```diff
@@ -1004,6 +1004,50 @@
...
+            "str_min": 60, "str_max": 210,
+            "sound": "glass breaking!",
+            "sound_fail": "whack!",
+            "sound_vol": 20,
+            "sound_fail_vol": 14,
+            "ter_set": "t_floor",
+            "items": [
+                { "item": "glass_shard", "count": [5, 10] },
+                { "item": "wire", "prob": 20 },
+				{ "item": "scrap", "count": [3, 5] }
```
Comment (illi-kun) [inline data/json/terrain.json]: please replace TABs by spaces here.

File: data/json/terrain.json [hunk 3]
```diff
@@ -1004,6 +1004,50 @@
...
+        }
+    },{
+        "type" : "terrain",
+        "id" : "t_reinforced_glass_shutter_open",
+        "name": "reinforced glass with open shutters",
+        "symbol": "LINE_OXOX",
+        "color": "ltcyan",
+        "move_cost": 0,
+        "flags": ["TRANSPARENT", "NOITEM", "WALL", "AUTO_WALL_SYMBOL", "OPENCLOSE_INSIDE"],
+		"close": "t_reinforced_glass_shutter",
```
Comment (illi-kun) [inline data/json/terrain.json]: please replace TABs by spaces here.

File: data/json/terrain.json [hunk 4]
```diff
@@ -1004,6 +1004,50 @@
...
+            "str_min": 40, "str_max": 210,
+            "sound": "glass breaking!",
+            "sound_fail": "whack!",
+            "sound_vol": 20,
+            "sound_fail_vol": 14,
+            "ter_set": "t_floor",
+            "items": [
+                { "item": "glass_shard", "count": [5, 10] },
+                { "item": "wire", "prob": 20 },
+				{ "item": "scrap", "count": [3, 5] }
```
Comment (illi-kun) [inline data/json/terrain.json]: please replace TABs by spaces here.

**Comment by chaosvolt** [comment]
Ah, interesting. Will have to keep an eye on this, as I sense terrain that will need spriting. o3o

**Pull request merged**